### DISCUSSION TITLE
Is there any common sense in using CannonJS in Enable3D?

### DISCUSSION THREAD

**Discussion by ProgrammingLife** [enable3d/enable3d#277] [General]
Is there a common sense to use Cannon in Enable3d? Since Ammo is too heavy engine which is too much for my simple games like platformers without using too much realistic physics. Or is it better to use Three.js+Cannon then?

**Comment by yandeu** [comment]
You can use three.js+cannon or enable3d+cannon.
It's up to your preference.

**Comment by brunoduartec** [comment]
I wat wandering use Cannon-es as physics engine, using the phaser-extension. Because Ammo seames to be very slow at chrome. Any advise on how to do it?

**Reply by yandeu** [reply]
You can easily add cannonjs. I'm sure you will find some three.js + cannon.js examples online.

> Ammo seames to be very slow at chrome

Really? Ammojs should be much faster.
